crates/astria-sequencer/src/service/consensus.rs

```diff
@@ -182,7 +206,9 @@ impl Consensus {
         })
     }
 
-    #[instrument(skip(self))]
+    #[instrument(skip_all, fields(
+        tx_hash = %telemetry::display::hex(&Sha256::digest(&deliver_tx.tx))
```

Comment:
Nit: see my other comment re abbreviation. It seems like `tx` is widely used in cometbft, so maybe it makes sense to use it here.